app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -1,8 +1,15 @@
 <p>If you’re studying for a degree and interested in a career in teaching, an internship could help you to understand what it’s really like in the classroom.</p>
 
-<p> You’ll get to spend 3 weeks in a school, gain new skills, and be paid £300 per week.</p>
+<p> You’ll get the opportunity to experience a range of activities to help you get a feel for school life. Internships last 3 weeks, start in June and you’ll be paid £300 per week.  </p>
```

Comment:
```suggestion
<p> You’ll get the opportunity to experience a range of activities to help you get a feel for school life. Internships last 3 weeks, start in June and you’ll be paid £300 per week.</p>
```
There are extra spaces at the end of this sentence - is this causing your linting error @philspray ?